RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+Hypothesis will now emit deprecation warnings if you use example() inside a
```

Comment:
Release notes are usually present tense, not future tense.  

I was also going to suggest making `example()` a link, but apparently this method is undocumented!  (ie mentioned in passing in the data page, but no linkable entry)  File under #774, I guess.

Reply:
I would argue that this *is* in the present tense. It describes the behaviour Hypothesis now has in a specific circumstance.

I don't mind changing the wording though.